neurom/features/__init__.py

```diff
@@ -88,7 +88,13 @@ def _get_feature_value_and_func(feature_name, obj, **kwargs):
     """
     feat = _find_feature_func(feature_name)
 
-    return np.array(list(feat(obj, **kwargs))), feat
+    res = feat(obj, **kwargs)
```

Comment:
Why this change? Tests still pass with the old code.

Reply:
See my two previous comments:
https://github.com/BlueBrain/NeuroM/pull/849#issuecomment-787809109
https://github.com/BlueBrain/NeuroM/pull/849#issuecomment-787846393

And there is a TODO at ``neurom/features/__init__.py::176`` so I guess this change was more or less planned.